dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/tasks/flink.py

```diff
@@ -85,6 +85,7 @@ def __init__(
         **kwargs
     ):
         super().__init__(name, TaskType.FLINK, *args, **kwargs)
+        self._resource = Resource()
```

Comment:
Could we init class `Resource` in property `main_jar`? Do too many things in `__init__` make us run slow and rasing the error rate.

Reply:
you are right. I will modify it.